src/commcare_cloud/fab/diff_templates/console.txt.j2

```diff
@@ -16,6 +16,19 @@ New version details:
 {{ mgs|warning }}
 {%- endfor %}
 
+{%- if changelogs.error -%}
+Changelogs:
+    Please review any pending changelogs from https://commcare-cloud.readthedocs.io/en/latest/ and make sure that these are applied on your environment before deploying to avoid getting your environment into a broaken state.
+
+{%- else -%}
+Changelogs:
+    There have been some changelogs since last deploy, you must make sure that these are applied on your environment before deploying to avoid getting your environment into a broaken state.
```

Comment:
@sravfeyn I'm seeing this notice when deploying staging, but I don't think it applies since no change logs are listed below. It looks like this is printed unconditionally, and I think that's a bug that was introduced in https://github.com/dimagi/commcare-cloud/commit/3391641f3e901eb3c781e5b1cf739a6050ca759a. Do you know why this is printed when there are no change logs, and is that the intended behavior?

Reply:
It looks like you have already addressed it, thank you!